src/main/java/org/openapitools/jackson/nullable/JsonNullableValueExtractorHelper.java

```diff
@@ -0,0 +1,30 @@
+package org.openapitools.jackson.nullable;
+
+import java.util.Collection;
+
+abstract class JsonNullableValueExtractorHelper {
+    public static void extractValues(JsonNullable<?> originalValue, IndexedValueSetter indexedValueSetter, ValueSetter valueSetter) {
```

Comment:
The paths for `javax` and `jakarta` are equal, so extract the logic and work with some functional interfaces to remove the redundancies.